ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1319,6 +1319,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-capability-additionalcaps
+- as: gcp-ipi-c3-metal-amd-f999
+  cron: 8 8 29 2 *
+  steps:
+    cluster_profile: gcp-qe-c3-metal
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-c3-metal-private
+- as: gcp-ipi-c3-metal-amd-f999-destructive
```

Comment:
We also need to add some z-stream upgrade ci jobs and y-stream upgrade ci jobs for the install workflow.

Reply:
I'm not sure if we need to add upgrade ci tests. @yunjiang29 @gpei WDYT?

Reply:
yeah, I think it would be a good idea to also test upgrade on this c3-metal profile, with the same frequency schedule as installation.

Reply:
OK, let me add for upgrading from 4.17 stable to 4.18 nightly. Thanks!